crates/query-engine/translation/src/translation/mutation/experimental/delete.rs

```diff
@@ -119,7 +115,7 @@ pub fn translate(
                 .iter()
                 .map(|by_column| {
                     let unique_key = arguments
-                        .get(&by_column.name)
+                        .get(&format!("{}{}", columns_prefix, by_column.name))
```

Comment:
Perhaps the `arguments` map should use a tuple or something as a key?

Reply:
arguments coming from ndc-spec, and they are not used only for keys, but also for `update_columns`, `pre_check`, etc.

Reply:
Got it, makes sense.